pkg/controller/migmigration/task.go

```diff
@@ -1362,6 +1379,66 @@ func (t *Task) hasImageStreams() (bool, error) {
 	return false, nil
 }
 
+// Get whether the associated plan has relevant artifacts using the analytics resource.
+func (t *Task) hasRelevantArtifacts() (bool, string, error) {
+	noMsg := ""
+	client, err := t.getDestinationClient()
+	if err != nil {
+		log.Trace(err)
+		return false, noMsg, err
+	}
+
+	analytic := migapi.MigAnalytic{}
+
+	err = client.Get(context.Background(),
+		k8sclient.ObjectKey{
+			Name:      t.PlanResources.MigPlan.Name,
+			Namespace: t.PlanResources.MigPlan.Namespace,
+		},
+		&analytic)
+	if err != nil {
+		log.Trace(err)
+		return false, noMsg, err
+	}
+
+	missingArtifactsInNamespace := make(map[string]string, 0)
+	for _, ns := range analytic.Status.Analytics.Namespaces {
+		missingArtifacts := make([]string, 0)
+		if ns.PVCount == 0 {
+			missingArtifacts = append(missingArtifacts, "Persistent Volumes")
+		}
```

Comment:
I don't think there's any need to check PV or Image count here, since I think those will be included in the `K8SResourceTotal` -- @djwhatle is this correct?

I think you only need to look at K8SResourceTotal here.

Reply:
I think @jmontleon would know the answer to this

Reply:
Looking closer, I think this may be an unintended usage of MigAnalytics, although I could certainly be missing some context.

Reply:
We include PV and Image counts in Analytics as @sseago mentioned (unless they are excluded from the migration either by flipping the booleans or adding the appropriate resources to the excludedResources list).